pages/partials/teams.html

```diff
@@ -0,0 +1,191 @@
+<!--
```

Comment:
this should be in a `templates/partials` dir at the root of the repo. That is searched in the setup as well but avoids the system trying to render it.